feat(protocol): Add command that start account deletion (#10480)
Scille/parsec-cloud: server/parsec/templates/account_email_deletion.txt

Hello!

> Please confirm your account deletion.

Reviewer: ```suggestion
You have requested to delete your Parsec Account.
```

> To confirm your account deletion:

Reviewer: ```suggestion
To confirm your Parsec Account deletion:
```

1. Make sure to download and install Parsec app (https://parsec.cloud/en/start-parsec)

> 2. Once installed, click on the link below to confirm your account deletion:

Reviewer: ```suggestion
2. Once installed, click on the link below to delete your Parsec Account:
```
